unified-codes/libs/data/src/lib/types/Entity.ts

```diff
@@ -56,6 +58,16 @@ export class Entity implements IEntity {
     return this._properties;
   }
 
+  getChild(code: string) {
+    const [entity] = this._children?.filter((child: IEntity) => child.code === code);
```

Comment:
Oh sorry - minor thing I just noticed from TS on this line (but it does still run fine):
`Type 'Entity[] | undefined' must have a '[Symbol.iterator]()' method that returns an iterator.`

and a similar one for getProperty

Reply:
Hmmm, c718a7985c9f22442b3f9637d0e0f64caced16f3 should fix that!